checkov/common/bridgecrew/platform_integration.py

```diff
@@ -123,6 +124,17 @@ def setup_api_urls(self) -> None:
     def is_prisma_integration(self) -> bool:
         return self.bc_api_key and not self.is_bc_token(self.bc_api_key)
 
+    @staticmethod
+    def is_token_valid(token: str) -> bool:
+        parts = token.split('::')
+        if len(parts) == 1:
+            return BcPlatformIntegration.is_bc_token(token)
+        elif len(parts) == 2:
```

Comment:
```suggestion
        parts = token.split('::')
        parts_len = len(parts)
        if parts_len == 1:
            return BcPlatformIntegration.is_bc_token(token)
        elif parts_len == 2:
```
no need to calculate it twice, if it is Prisma key